src/Disks/S3/DiskS3.cpp

```diff
@@ -0,0 +1,1147 @@
+#include "DiskS3.h"
```

Comment:
This file huge changes looks wrong. How this happened?

Reply:
many thx your review, it is my local file , already delete it.